build.py

```diff
@@ -377,31 +377,6 @@ def file_lookup(name):
 
       code = HEADER + "\n" + json_data["compiledCode"]
       code = code.replace(remove, "")
-
-      # Trim down Google's Apache licences.
-      # The Closure Compiler used to preserve these until August 2015.
-      # Delete this in a few months if the licences don't return.
```

Comment:
It has been a few months.  The compiler's brief experiment with preserving licenses seems to be truly dead.  Good.